app/data-sources/rural-payments/RuralPayments.js

```diff
@@ -22,54 +21,80 @@ export class RuralPayments extends RESTDataSource {
   }
 
   async fetch (path, incomingRequest) {
-    logger.verbose('#RuralPayments - new request', {
+    const request = {
+      method: incomingRequest.method.toUpperCase(),
       path,
-      method: incomingRequest.method
-    })
+      payload: incomingRequest.body,
+      headers: incomingRequest.headers
+    }
 
     const doRequest = async (count = 1) => {
       try {
+        this.logger.verbose('#datasource - Rural payments - request', {
+          request,
+          code: RURALPAYMENTS_API_REQUEST_001
+        })
+
+        this.logger.verbose('#datasource - apim - request', {
+          request,
+          code: APIM_APIM_REQUEST_001
+        })
+
         const requestStart = Date.now()
         const result = await super.fetch(path, incomingRequest)
-        const requestEnd = (Date.now() - requestStart)
-        logger.verbose('#RuralPayments - response ', {
-          path,
-          status: result.response?.status
-        })
-        logger.health('#RuralPayments - request success', {
-          code: RURALPAYMENTS_REQUEST_ALL_001,
-          requestTimeMs: requestEnd
+        const requestTimeMs = (Date.now() - requestStart)
+
+        this.logger.debug('#datasource - Rural payments - response', {
+          request,
+          response: {
+            status: result.response?.status
+          },
+          code: RURALPAYMENTS_API_REQUEST_001,
+          requestTimeMs
         })
-        logger.health('#APIM - request success', {
-          code: APIM_REQUEST_RURAL_PAYMENTS_REQUEST_001
+
+        this.logger.debug('#datasource - apim - response', {
+          request,
+          response: {
+            status: result.response?.status
+          },
+          code: APIM_APIM_REQUEST_001,
+          requestTimeMs
         })
+
         return result
       } catch (error) {
         // Handle occasionally 500 error produced by APIM
         // TODO: Once APIM has been fixed, remove retry logic
         if (error?.extensions?.response?.status === StatusCodes.INTERNAL_SERVER_ERROR && count < maximumRetries) {
-          logger.warn(`#RuralPayments - Error, Retrying request (${count})`, { error })
+          this.logger.warn('#datasource - apim - retrying request', { request, error, code: APIM_APIM_REQUEST_001, retryCount: count })
           return doRequest(count + 1)
         }
-        // if response is text, then the error is from RoralPayments
-        if (error?.extensions?.response?.headers?.get('Content-Type')?.includes('text/html')) {
-          logger.error('#RuralPayments - error', {
+
+        if (error?.extensions?.response?.status === StatusCodes.FORBIDDEN) {
```

Comment:
Is this an APIM error if so they are our credentials so should always have access?

Reply:
If APIM credentials are wrong would be good to know if this is hit.

Reply:
no sorry its for rural payments, have updated the message